internal/providers/docker/components/container/mount.go

```diff
@@ -0,0 +1,61 @@
+package container
+
+import (
+	"fmt"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/docker/api/types/mount"
+)
+
+func makeMountFromVolumeString(workspaceRoot, volume string) (mount.Mount, error) {
+	// This obviously doesn't handle colons in the path well but it would appear
+	// that docker-compose doesn't handle those either.
+	volumeParts := strings.Split(volume, ":")
+	if len(volumeParts) > 3 {
```

Comment:
In order to handle colons in paths, would the correct thing to do here be to do `strings.LastIndex(volume, ":")` and split at that index?

Reply:
I don't think that fixes it since the separating colon appears between two file paths. So splitting on the last colon also does the wrong thing for `./app:/du:mb/filepath`.